Sources/MapboxDirections/Incident.swift

```diff
@@ -21,46 +20,215 @@ public struct Incident: Codable, Equatable {
+        case leftCenter = "LEFT CENTER"
+        /// Left turn lane
+        case leftTurnLane = "LEFT TURN LANE"
+        /// Center lane
+        case center = "CENTER"
+        /// Right lane
+        case right = "RIGHT"
+        /// Right and center lanes
+        case rightCenter = "RIGHT CENTER"
+        /// Right turn lane
+        case rightTurnLane = "RIGHT TURN LANE"
+        /// High occupancy vehicle lane
+        case highOccupancyVehicle = "HOV"
+        /// Side lane
+        case side = "SIDE"
+        /// Shoulder lane
+        case shoulder = "SHOULDER"
+        /// Median lane
+        case median = "MEDIAN"
+        /// 1st Lane.
+        ///
+        /// Mind the driving side.
+        case lane1 = "1"
+        /// 2nd Lane.
+        ///
+        /// Mind the driving side.
+        case lane2 = "2"
+        /// 3rd Lane.
+        ///
+        /// Mind the driving side.
+        case lane3 = "3"
+        /// 4th Lane.
+        ///
+        /// Mind the driving side.
+        case lane4 = "4"
+        /// 5th Lane.
+        ///
+        /// Mind the driving side.
+        case lane5 = "5"
+        /// 6th Lane.
+        ///
+        /// Mind the driving side.
+        case lane6 = "6"
+        /// 7th Lane.
+        ///
+        /// Mind the driving side.
+        case lane7 = "7"
+        /// 8th Lane.
+        ///
+        /// Mind the driving side.
+        case lane8 = "8"
+        /// 9th Lane.
+        ///
+        /// Mind the driving side.
+        case lane9 = "9"
+        /// 10th Lane.
+        ///
+        /// Mind the driving side.
+        case lane10 = "10"
+    }
+    
+    /// Incident identifier
+    public var identifier: String
+    /// The kind of an incident
+    ///
+    /// This value is set to `nil` if `kind` value is not supported.
+    public var kind: Kind?
+    var rawKind: String
+    /// Short description of an incident. May be used as an additional info.
+    public var description: String
+    /// Date when incident item was created.
+    public var creationDate: Date
+    /// Date when incident happened.
+    public var startDate: Date
+    /// Date when incident shall end.
+    public var endDate: Date
+    /// Shows severity of an incident. May be not available for all incident types.
+    public var impact: String?
+    /// Provides additional classification of an incident. May be not available for all incident types.
+    public var subtype: String?
+    /// Breif description of the subtype. May be not available for all incident types and is not available if `subtype` is `nil`
+    public var subtypeDescription: String?
+    /// Contains list of ISO 14819-2:2013 codes
+    ///
+    /// See https://www.iso.org/standard/59231.html for details
+    public var alertCodes: Set<Int>
+    /// A list of lanes indices, affected by the incident
+    ///
+    /// `nil` value indicates that such lane identifier is not supported
+    public var lanesBlocked: Set<BlockedLane?>
```

Comment:
As tail work, maybe we should consider turning BlockedLane into an OptionSet to more seamlessly accommodate multiple values.